src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,138 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> tail;
+    private Node<T> head;
+    private Node<T> currentNode;
```

Comment:
Why do you need this file? Do you need access to the value from more than one method?